### ISSUE TITLE
Change intro-slide page points

### ISSUE THREAD

**Issue by simomps** [digitalfabrik/entitlementcard#2728]
### Describe the Problem

Intro-slides: the points at the bottom of the page have no alt text 
but are selectable by screenreader or a keyboard focus 

### Describe Your Preferred Solution

Add alt-text "slide 1 of4", "slide 2 of 4" to each slide
Make all points one single item to select

### Describe Possible Alternatives

Make items not selectable and use them only as a visual navigation

### Related Issues

https://github.com/digitalfabrik/entitlementcard/issues/2716

### Additional Information

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/63ab7096-6d42-4c1e-a3fb-8730379265f0" />
